# HG changeset patch
# User Torsten <ttl@justmail.de>
# Date 1372490503 -7200
#      Sat Jun 29 09:21:43 2013 +0200
# Node ID a86327a7d9fb28e06081b8355ffed685117f7032
# Parent  06657ee17aa2e127edd74f2597062561bb0a4490
fix of typos in text strings of the gui

* settings-dialog.ui, settings-dialog.cc(read_lexer_settings),
  octave-dock-widget.cc(make_widget), files-dock-widget.cc(contextmenu_delete)

diff --git a/libgui/src/files-dock-widget.cc b/libgui/src/files-dock-widget.cc
--- a/libgui/src/files-dock-widget.cc
+++ b/libgui/src/files-dock-widget.cc
@@ -500,17 +500,17 @@ files_dock_widget::contextmenu_delete (b
 
   for( QModelIndexList::iterator it = rows.begin (); it != rows.end (); it++)
     {
       QModelIndex index = *it;
 
       QFileInfo info = _file_system_model->fileInfo(index);
 
       if(QMessageBox::question(this, tr("Delete file/directory"), 
-                               tr("Are you sre you want to delete\n") + info.filePath(),
+                               tr("Are you sure you want to delete\n") + info.filePath(),
                                QMessageBox::Yes|QMessageBox::No) == QMessageBox::Yes) 
         {
            if(info.isDir())
              {
                // see if direcory is empty
                QDir path(info.absoluteFilePath());
                QList<QFileInfo> fileLst = path.entryInfoList(QDir::AllEntries | QDir::NoDotAndDotDot);
 
diff --git a/libgui/src/octave-dock-widget.cc b/libgui/src/octave-dock-widget.cc
--- a/libgui/src/octave-dock-widget.cc
+++ b/libgui/src/octave-dock-widget.cc
@@ -164,17 +164,17 @@ octave_dock_widget::make_widget ()
   _parent->addDockWidget (static_cast<Qt::DockWidgetArea> (area), this);
 
   // FIXME: restoreGeometry is ignored for docked widgets and its child widget
   // restoreGeometry (settings->value
   //        ("DockWidgets/" + objectName ()).toByteArray ());
 
   // adjust the (un)dock icon
   _dock_action->setIcon (QIcon (":/actions/icons/widget-undock.png"));
-  _dock_action->setToolTip (tr ("Unock widget"));
+  _dock_action->setToolTip (tr ("Undock widget"));
 }
 
 // slot for (un)dock action
 void
 octave_dock_widget::change_floating (bool)
 {
   if (parent())
     {
diff --git a/libgui/src/settings-dialog.cc b/libgui/src/settings-dialog.cc
--- a/libgui/src/settings-dialog.cc
+++ b/libgui/src/settings-dialog.cc
@@ -262,17 +262,17 @@ settings_dialog::read_lexer_settings (Qs
         }
       else   // other styles
         {
           select_font[i]->setCurrentFont (actual_font);
           if (actual_font.family () == default_font.family ())
             select_font[i]->setEditText (lexer->description (0));
           font_size[i]->setRange (-4,4);
           font_size[i]->setValue (actual_font.pointSize ()-default_size);
-          font_size[i]->setToolTip (tr ("Difference to the defalt size"));
+          font_size[i]->setToolTip (tr ("Difference to the default size"));
           if (lexer->paper (styles[i]) == default_color)
             bg_color[i] = new color_picker (dummy_color);
           else
             bg_color[i] = new color_picker (lexer->paper (styles[i]));
             bg_color[i]->setToolTip
                   (tr ("Background color, pink (255,0,255) means default"));
         }
       attrib_font[0+3*i] = new QCheckBox (tr("b"));
diff --git a/libgui/src/settings-dialog.ui b/libgui/src/settings-dialog.ui
--- a/libgui/src/settings-dialog.ui
+++ b/libgui/src/settings-dialog.ui
@@ -148,17 +148,17 @@
               <property name="text">
                <string>Letter icons</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="general_icon_graphic">
               <property name="text">
-               <string>Graphic  icons</string>
+               <string>Graphic icons</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_6">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
